src/main/java/org/folio/circulation/resources/DeclareClaimedReturnedItemAsMissingResource.java

```diff
@@ -23,42 +31,70 @@
 
 public class DeclareClaimedReturnedItemAsMissingResource extends Resource {
 
+  private static final String NOTE_MESSAGE = "Claimed returned item marked missing";
+  private static final String NOTE_DOMAIN  = "loans";
+
   public DeclareClaimedReturnedItemAsMissingResource(HttpClient client) {
     super(client);
   }
 
   @Override
   public void register(Router router) {
     new RouteRegistration("/circulation/loans/:id/declare-claimed-returned-item-as-missing", router)
-      .create(this::declareClaimedReturnedItemAsMissing);
+        .create(this::declareClaimedReturnedItemAsMissing);
   }
 
   private void declareClaimedReturnedItemAsMissing(RoutingContext routingContext) {
     final WebContext context = new WebContext(routingContext);
 
-    createRequest(routingContext)
-      .after(request -> processDeclareClaimedReturnedItemAsMissing(routingContext, request))
-      .thenApply(r -> r.toFixedValue(NoContentResponse::noContent))
-      .thenAccept(context::writeResultToHttpResponse);
+    createRequest(routingContext).after(request -> processDeclareClaimedReturnedItemAsMissing(routingContext, request))
+        .thenApply(r -> r.toFixedValue(NoContentResponse::noContent)).thenAccept(context::writeResultToHttpResponse);
   }
 
-  private CompletableFuture<Result<Loan>> processDeclareClaimedReturnedItemAsMissing(
-    RoutingContext routingContext, ChangeItemStatusRequest request) {
-
+  private CompletableFuture<Result<Loan>> processDeclareClaimedReturnedItemAsMissing(RoutingContext routingContext,
+      ChangeItemStatusRequest request) {
     final Clients clients = Clients.create(new WebContext(routingContext), client);
-    final ChangeItemStatusService changeItemStatusService =
-      new ChangeItemStatusService(clients);
+    final ChangeItemStatusService changeItemStatusService = new ChangeItemStatusService(clients);
 
     return changeItemStatusService.getOpenLoan(request)
-      .thenApply(NotInItemStatusValidator::refuseWhenItemIsNotClaimedReturned)
-      .thenApply(loanResult -> declareLoanMissing(loanResult, request))
-      .thenCompose(changeItemStatusService::updateLoanAndItem);
+        .thenApply(NotInItemStatusValidator::refuseWhenItemIsNotClaimedReturned)
+        .thenApply(loanResult -> declareLoanMissing(loanResult, request))
+        .thenCompose(changeItemStatusService::updateLoanAndItem)
+        .thenCompose(loanResult -> loanResult.after(loan -> createNote(clients, loan)));
```

Comment:
I'm ok with deeper indentation, I would prefer if it was consistent, at least in the same file